build.gradle.kts

```diff
@@ -53,6 +53,7 @@ dependencies {
     testCompile(project(":virtual-users"))
     testCompile(fileTree(mapOf("dir" to "lib", "include" to "*.jar")))
     testCompile("com.atlassian.performance.tools:jira-performance-tests:[3.3.0,4.0.0)")
+    testCompile("com.atlassian.performance.tools:aws-infrastructure:[2.10.0,3.0.0)")
```

Comment:
A tooling question.
If this reference wasn't specified here previously were the entries in the lockfiles there because they were inherited via another dependency?

Reply:
* it wasn't specified here, because this is branched off of an experimental branch
* lockfiles do materialize the entire dependency tree, including transitive ones